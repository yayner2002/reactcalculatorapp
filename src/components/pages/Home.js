import React from 'react';

function Home() {
  return (
    <div />
  );
}

export default Home;
