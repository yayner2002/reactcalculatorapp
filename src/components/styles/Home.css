.homeContainer {
  width: 90%;
  box-sizing: border-box;
  margin: 2rem auto;
}
