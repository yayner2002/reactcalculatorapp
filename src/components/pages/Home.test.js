import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('correctlly renders a component', () => {
  test('should render home component', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
