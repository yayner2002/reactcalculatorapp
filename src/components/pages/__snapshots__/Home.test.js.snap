// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctlly renders a component should render home component 1`] = `
<div
  className="homeContainer"
>
  <h1>
    Well come to our page
  </h1>
  <section>
    <h2>
      What we do
    </h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa iste aliquid id, earum itaque repellat commodi sint omnis tenetur veniam sit. Libero nihil rerum sed recusandae eius, dolorum illum? Praesentium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa iste aliquid id, earum itaque repellat commodi sint omnis tenetur veniam sit. Libero nihil rerum sed recusandae eius, dolorum illum? Praesentium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa iste aliquid id, earum itaque repellat commodi sint omnis tenetur veniam sit. Libero nihil rerum sed recusandae eius, dolorum illum? Praesentium.
    </p>
  </section>
  <section>
    <h2>
      What we are looking for
    </h2>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa iste aliquid id, earum itaque repellat commodi sint omnis tenetur veniam sit. Libero nihil rerum sed recusandae eius, dolorum illum? Praesentium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa iste aliquid id, earum itaque repellat commodi sint omnis tenetur veniam sit. Libero nihil rerum sed recusandae eius, dolorum illum? Praesentium. Lorem ipsum dolor sit amet consectetur adipisicing elit. Culpa iste aliquid id, earum itaque repellat commodi sint omnis tenetur veniam sit. Libero nihil rerum sed recusandae eius, dolorum illum? Praesentium.
    </p>
  </section>
</div>
`;
